feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -360,7 +505,17 @@ class PreviewViewModel @AssistedInject constructor(
 
     fun setImageFormat(imageFormat: ImageOutputFormat) {
         viewModelScope.launch {
-            cameraUseCase.setImageFormat(imageFormat)
+            cameraUseCase.setImageFormat(imageFormat).also {
+                _previewUiState.update { old ->
+                    (old as? PreviewUiState.Ready)?.copy(
+                        captureModeToggleUiState =
+                        getCaptureToggleUiState(
+                            old.systemConstraints,
+                            cameraUseCase.getCurrentSettings().value!!
+                        )
+                    ) ?: old
+                }
+            }
```

Comment:
This `also` block shouldn't be needed. The updates from the CameraUseCase should be sufficient.